src/ManagementRoomOutput.ts

```diff
@@ -84,6 +98,32 @@ export default class ManagementRoomOutput {
         return content;
     }
 
+    /**
+     * Log an error message to the management room, console and Sentry.
+     */
+    public async logException(level: LogLevel, mjolnir: Mjolnir | null, module: string, error: Error, extras: Record<string, string | number> = {}, isRecursive = false) {
+        let sentryLevel: SeverityLevel | undefined;
+        if (level === LogLevel.DEBUG) {
+            sentryLevel = 'debug';
+        } else if (level === LogLevel.ERROR) {
+            sentryLevel = 'error';
+        } else if (level === LogLevel.INFO) {
+            sentryLevel = 'info';
+        } else if (level === LogLevel.WARN) {
+            sentryLevel = 'warning';
+        }
+        let scope = {
+            extra: {
+                module,
+                mjolnirUserId: this.mjolnirUserId,
+                ...extras
+            },
+            level: sentryLevel
+        };
+        Sentry.captureException(error, scope);
+        await this.logMessage(level, module, error.message, null, isRecursive);
+    }
+
```

Comment:
Honestly, we should really implement this over the matrix-bot-sdk [logger](https://github.com/turt2live/matrix-bot-sdk/blob/main/src/logging/LogService.ts#L84-L90) and combine it with our existing logger (the `RichConsoleLogger`).

Reply:
I know i requested for the mjolnirUserId to be logged, and that would seem to conflict with sentry reporting to go into the logger, but you can do this using [scopes](https://docs.sentry.io/platforms/javascript/enriching-events/scopes/). I also don't  see the `mjolnirUserId` appearing anywhere in the web interface, i think it either has to be [user info](https://docs.sentry.io/platforms/javascript/enriching-events/identify-user/) (that we should probably reserve for who sent a command etc) or a tag.

Reply:
It's just not right that we would only use this api with the extra information explicitly.